pauthabielf64/pauthabielf64.rst

```diff
@@ -537,23 +509,19 @@ Example code for the PLT sequence generated by the static linker.
     autia1716
     br   x17
 
-With the Armv8.3-a extension the autia1716 and br x17 can be combined
-into a single instruction braa x17, x16
+If instructions not encoded in the hint space can be used it is
+possible to combine the autia1716 and br x17 into a single instruction
```

Comment:
.. single instruction`:` bra ..

fixed font for instructions?, also below..

Reply:
ACK, will fix.